fastlane.gemspec

```diff
@@ -71,6 +71,9 @@ Gem::Specification.new do |spec|
   # If you upgrade this gem, make sure to upgrade the users of it as well.
   spec.add_dependency 'google-api-client', '>= 0.13.1', '< 0.14.0' # Google API Client to access Play Publishing API
 
+  # jwt 2.x requires Ruby 2.1. This is ultimately pulled in by the google-api-client.
+  spec.add_dependency 'jwt', '~> 1.5'
```

Comment:
Instead of using `~>` could we use `'>= 1.5', < 3.0.0` to allow 1.5, but use a 2.x.x version if possible/available?

Reply:
The requirements from googleauth and signet are already similar:

https://rubygems.org/gems/googleauth
https://rubygems.org/gems/signet

This is what happens currently:

```
[jdee@Jimmy-Dees-MacBookPro ~]$ rvm use 2.0.0
Using /Users/jdee/.rvm/gems/ruby-2.0.0-p648
[jdee@Jimmy-Dees-MacBookPro ~]$ gem uninstall jwt

You have requested to uninstall the gem:
	jwt-1.5.6

googleauth-0.6.2 depends on jwt (< 3.0, >= 1.4)
signet-0.8.1 depends on jwt (< 3.0, >= 1.5)
If you remove this gem, these dependencies will not be met.
Continue with Uninstall? [yN]  y
Successfully uninstalled jwt-1.5.6
[jdee@Jimmy-Dees-MacBookPro ~]$ gem install fastlane
Fetching: jwt-2.1.0.gem (100%)
ERROR:  Error installing fastlane:
	The last version of jwt (< 3.0, >= 1.5) to support your Ruby & RubyGems was 1.5.6. Try installing it with `gem install jwt -v 1.5.6` and then running the current command again
	jwt requires Ruby version >= 2.1. The current ruby version is 2.0.0.
```

I'm afraid the gemspec actually has to pin, which is unfortunate. I'd love to find a better way to do it.

Reply:
Now I'm skeptical about this filter on the `jwt` version. The problem is that someone may have a newer version of `jwt` installed with Ruby >= 2.1. Then with this `gem install fastlane` may fail because RubyGems will refuse to active 1.5.6 because a newer version is already active.

There may be no graceful solution to the `gem install` issue. It may be a matter of documenting:

1. Use Ruby >= 2.1, preferably with a version manager.
2. Use a Gemfile whenever possible.
3. If you can't do either of these things, then with 2.0, run `gem install jwt -v 1.5.6` before `gem install fastlane`.

I think 2.0 is just becoming harder to support.